### PR TITLE
Dependencies versions update

### PR DESCRIPTION
**Pull request by senivam**
Signed-off-by: Maxim Nesen <maxim.nesen@oracle.com>

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -2193,17 +2182,28 @@
...
         <jsonp.ri.version>1.1.6</jsonp.ri.version>
         <jsonp.jaxrs.version>1.1.6</jsonp.jaxrs.version>
+        <jsp.version>2.3.6</jsp.version>
+        <jstl.version>1.2.7</jstl.version>
+        <jta.api.version>1.3.3</jta.api.version>
+        <microprofile.config.version>2.0.1</microprofile.config.version>
         <mimepull.version>1.9.13</mimepull.version>
         <moxy.version>2.7.9</moxy.version>
-        <yasson.version>1.0.9</yasson.version>
+        <osgi.version>6.0.0</osgi.version>
```
[the remark this answers is not in the document]
Comment (senivam) [reply pom.xml:2200]: osgi is Eclipse project

[the remark this answers is not in the document]
Comment (senivam) [reply pom.xml]: sorry, only relevant to org.osgi.service.cm. The rest of the versions is moved back to the place.

File: pom.xml [hunk 2]
```diff
@@ -2193,17 +2182,28 @@
...
         <jetty.servlet.api.25.version>6.1.14</jetty.servlet.api.25.version>
         <jsonb.api.version>1.0.2</jsonb.api.version>
         <jsonp.ri.version>1.1.6</jsonp.ri.version>
         <jsonp.jaxrs.version>1.1.6</jsonp.jaxrs.version>
+        <jsp.version>2.3.6</jsp.version>
+        <jstl.version>1.2.7</jstl.version>
+        <jta.api.version>1.3.3</jta.api.version>
+        <microprofile.config.version>2.0.1</microprofile.config.version>
         <mimepull.version>1.9.13</mimepull.version>
         <moxy.version>2.7.9</moxy.version>
```
[the remark this answers is not in the document]
Comment (senivam) [reply pom.xml]: in central, there is only 2.7.10

**Review by jansupol — APPROVED** [review]

**Review by tomas-langer — APPROVED** [review]
LGTM

**Pull request merged**